web/.eslintrc.json

```diff
@@ -14,7 +14,8 @@
     ]
   },
   "env": {
-    "browser": true
+    "browser": true,
+    "jest": true
```

Comment:
Do you think this `jest` env should be in the `overrides` section?  Something like...

```json
{
  "overrides": [
    {
      "files": ["**/*.test.js"],
      "env": {
        "jest": true
      }
    }
  ]
}
```

I think we shouldn't need the `globals` in there with the `env` set.  I like this `env` version way better.  😛